src/Workspaces/Core/Portable/CodeActions/Operations/OpenDocumentOperation.cs

```diff
@@ -10,24 +10,17 @@ namespace Microsoft.CodeAnalysis.CodeActions
     /// <summary>
     /// A code action operation for requesting a document be opened in the host environment.
     /// </summary>
-    public sealed class OpenDocumentOperation : CodeActionOperation
+    public sealed class OpenDocumentOperation(DocumentId documentId, bool activateIfAlreadyOpen = false) : CodeActionOperation
     {
-        private readonly DocumentId _documentId;
-        private readonly bool _activate;
-
-        public OpenDocumentOperation(DocumentId documentId, bool activateIfAlreadyOpen = false)
-        {
-            _documentId = documentId ?? throw new ArgumentNullException(nameof(documentId));
-            _activate = activateIfAlreadyOpen;
-        }
+        private readonly DocumentId _documentId = documentId ?? throw new ArgumentNullException(nameof(documentId));
```

Comment:
❔ How does it know when to create a backing field for the parameter, and when to not create it?

Reply:
:arrow_right: Answered as part of https://github.com/dotnet/roslyn/pull/68299/files#r1203256887